examples/dataset_generation/create_alpaca_dataset.ipynb

```diff
@@ -8,106 +8,97 @@
     "# Generate dataset for ALPACA\n",
     "\n",
     "## Use token from OpenAI to generate dataset for finetuning\n",
-    "### You can get token from [here](https://beta.openai.com/docs/api-reference/authentication)"
+    "### You can get token from [here](https://beta.openai.com/docs/api-reference/authentication)\n",
+    "### You can also use AI Studio and Cohere to generate dataset"
    ]
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "WARNING: CUDA is not available, using CPU instead, can be very slow\n"
-     ]
+   "execution_count": null,
+   "metadata": {
+    "pycharm": {
+     "is_executing": true
     }
-   ],
+   },
+   "outputs": [],
    "source": [
     "from xturing.datasets import InstructionDataset\n",
-    "from xturing.model_apis import openai"
+    "from xturing.model_apis import Davinci"
```

Comment:
I don't like this. What if Cohere also has a model called Davinci? We should reflect the change properly. I would prefer something like:

```from xturing.model_apis.openai import Davinci```

Reply:
Wait why is this different from `TextGenerationAPI` mentioned below?

Reply:
Ok, it seems you just need to fix this, right?

Reply:
TextGenerationAPI is base class, and other api classes are using it